data/uhc_pack/functions/running/diamond_chestplate/cleanup.mcfunction

```diff
@@ -0,0 +1,4 @@
+clear @s minecraft:diamond_chestplate
+give @s minecraft:diamond 8
+
+scoreboard players set @s uhcDiaChest 0
```

Comment:
I would like to see some text shown to the user and a sound effect played, ideally one that sounds like something crumbling. Take a look at `data/uhc_pack/functions/running/suspicious_stew/replace.mcfunction` for an example of what it should look like.

Reply:
Ah yes, that's a great idea :)